src/routes/JobViewRoute.js

```diff
@@ -35,23 +37,33 @@ class JobViewRoute extends React.Component {
     }).isRequired,
   };
 
-  state = { showConfirmDelete: false };
+  constructor(props) {
+    super(props);
+
+    this.callout = React.createRef();
+    this.state = { showConfirmDelete: false };
+  }
+
+  componentDidUpdate(prevProps) {
```

Comment:
This can go away if we are handling the create logic in the `componentDidUpdate` of jobsRoute as mentioned above.

Reply:
Moved